src/mpl/configure.ac

```diff
@@ -942,6 +942,14 @@ AC_SUBST(libmpl_so_versionflags)
 
 #######################################################################
 
+#######################################################################
+## START OF GPU CODE
+#######################################################################
+
+
+#######################################################################
+## END OF GPU CODE
+#######################################################################
```

Comment:
Unnecessary code?

Reply:
This is addressed since I need to add the  AC_DEFINE for the fallback when no GPU support is detected.

Reply:
Why do we need this?   When the other GPU types are not defined, it is already the fallback path.   We don't need yet another definition for it.